File: rolling-update-kubernetes/src/test/scala/doc/akka/rollingupdate/kubernetes/AppVersionRevisionCompileOnly.scala
/*
 * Copyright (C) 2017-2025 Lightbend Inc. <https://akka.io>
 */

package doc.akka.rollingupdate.kubernetes

import akka.actor.ActorSystem
import akka.rollingupdate.kubernetes.AppVersionRevision

object AppVersionRevisionCompileOnly {

  val system = ActorSystem()

  //#start
  // Starting the AppVersionRevision extension
  // preferred to be called before ClusterBootstrap
  AppVersionRevision(system).start()
  //#start

}

File: rolling-update-kubernetes/src/test/java/jdoc/akka/rollingupdate/kubernetes/AppVersionRevisionCompileOnly.java
/*
 * Copyright (C) 2017-2025 Lightbend Inc. <https://akka.io>
 */

package jdoc.akka.rollingupdate.kubernetes;

import akka.actor.ActorSystem;
import akka.rollingupdate.kubernetes.AppVersionRevision;
public class AppVersionRevisionCompileOnly {
    public static void bootstrap() {

        ActorSystem system = ActorSystem.create();

        //#start
        // Starting the AppVersionRevision extension
        // preferred to be called before ClusterBootstrap
        AppVersionRevision.get(system).start();
        //#start
    }
}
